src/Store/StoreThread.re

```diff
@@ -66,6 +66,7 @@ let registerCommands = (~dispatch, commands) => {
 
 let start =
     (
+      ~showUpdateChangelog=true,
```

Comment:
I really don't like these dirty hacks being scattered all over the place just for the sake of testing, with very little indication of their actual purpose. Making the code more convoluted and prone to error really undermines the point of writing tests in the first place. IMO, doing it properly could be mocking out the entire store folder, or the persistence mechanism, for example.

Reply:
> IMO, doing it properly could be mocking out the entire store folder, or the persistence mechanism, for example.

@glennsl - That would be great! It would've be nice to have this in the initial persistence implementation (along with tests). Would you be up for adding this?

Reply:
Ideally we could tackle it holistically, and address the `configurationFilePath` and `keybindingsFilePath` too: https://github.com/onivim/oni2/blob/599e880bda42adeb9b69318c73ef30c77036de48/src/Store/StoreThread.re#L70 just having a way to specify a root config folder. That would help significantly.

Reply:
In the meantime, I'm going to merge this so I can unblock the next round of fixes.